lms/static/sass/shared/_course_object.scss

```diff
@@ -7,33 +7,44 @@
       width: flex-grid(4);
       @include margin-right(flex-gutter());
       float: left;
-
-      &:nth-child(3n+3) {
-        @include margin-right(0);
-      }
     }
 
     .courses-listing {
+      @include outer-container;
       @include clearfix();
```

Comment:
I don't believe the ``clearfix()`` mixin is needed anymore with the ``outer-container`` being applied (I think it clears the document flow as part of its role). Please remove.